internal/test/scorecard/tests.go

```diff
@@ -288,3 +288,89 @@ func logEvents(r *scapiv1alpha3.TestResult, client *CryostatClientset, namespace
 	}
 	return nil
 }
+
+func newCryostatCR(namespace string, withIngress bool) *operatorv1beta1.Cryostat {
+	cr := &operatorv1beta1.Cryostat{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cryostat-cr-test",
+			Namespace: namespace,
+		},
+		Spec: operatorv1beta1.CryostatSpec{
+			Minimal:           false,
+			EnableCertManager: &[]bool{true}[0],
+		},
+	}
+
+	if withIngress {
+		pathType := netv1.PathTypePrefix
+		cr.Spec.NetworkOptions = &operatorv1beta1.NetworkConfigurationList{
+			CoreConfig: &operatorv1beta1.NetworkConfiguration{
+				Annotations: map[string]string{
+					"nginx.ingress.kubernetes.io/backend-protocol": "HTTPS",
+				},
+				IngressSpec: &netv1.IngressSpec{
+					TLS: []netv1.IngressTLS{},
```

Comment:
Try adding an empty `IngressTLS` struct here and for the `GrafanaConfig` as well. I think the operator uses the length of the TLS slice to distinguish between HTTP and HTTPS. We'd like to test with HTTPS if possible.